index.html

```diff
@@ -152,6 +152,9 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 The Working Group will also seek consensus around DID Method specifications, and
 may produce related documents.
         </p>
+        <p>
+The Working Group will begin working toward a specification for DID Resolution.
```

Comment:
```suggestion
The Working Group will begin working toward a specification for DID Resolution (incl. DID URL Dereferencing).
```

Reply:
addressed in [409f1c6](https://github.com/w3c/did-wg-charter/pull/25/commits/409f1c61b8000805fe33cb67e8e8ee43ad8bf25a)